runtime-builder/go-1.11.yaml

```diff
@@ -1,6 +1,4 @@
 steps:
 - name: 'gcr.io/gcp-runtimes/go1-builder:1.11'
-- name: 'gcr.io/cloud-builders/docker:latest'
-  args: ['build', '-t', '$_OUTPUT_IMAGE', '.']
-images:
-- '$_OUTPUT_IMAGE'
+- name: 'gcr.io/kaniko-project/executor:v0.4.0@sha256:0bbaa4859eec9796d32ab45e6c1627562dbc7796e40450295b9604cd3f4197af'
```

Comment:
Google Cloud Build (by Google Cloud Build) doesn't like the fancy tag/digest syntax.